ink-wrapper-types/src/aleph_client.rs

```diff
@@ -119,7 +129,7 @@ impl crate::UploadConnection<TxInfo, anyhow::Error> for aleph_client::SignedConn
                 call.wasm,
                 None,
                 Determinism::Deterministic,
-                TxStatus::Finalized,
+                call.tx_status.into(),
```

Comment:
I don't know if it makes sense for _read_ calls - they don't modify the chain anyway so the data they return is valid as good as it can be.

Reply:
Ah, GH collaped the code and I misread - this is _upload_ and not _read_...

Reply:
I think it might make sense if we ever add the option to execute a read call, like we discussed, but not for now.